research-hub-web/cypress/integration/homepage.e2e.js

```diff
@@ -10,33 +9,27 @@ describe('ResearchHubs Homepage', () => {
 
     // Only enable if featured items are enabled
     it('displays featured articles', () => {
-        cy.expect('h2.featured-title').not.to.be.empty;
-        cy.expect('.featured-content app-cards mat-nav-list:first-child').not.to.be.empty;
+        cy.get('app-content-title h2').text().should('not.be.empty');
+        cy.get('app-cards mat-nav-list:first-child h4 a').text().should('not.be.empty');
     })
 
     it('displays research categories', () => {
-        cy.expect('#research-categories h2').not.to.be.empty;
-        cy.expect('#research-categories mat-card').not.to.be.empty;
+        cy.get('#research-categories h2').text().should('not.be.empty');
+        cy.get('#research-categories mat-card').text().should('not.be.empty');
     })
 
     it('displays research activities', () => {
-        cy.expect('#research-activities h2').not.to.be.empty;
-        cy.expect('#research-activities .inner-panel h3').not.to.be.empty;
-        cy.expect('#research-activities .inner-panel p').not.to.be.empty;
+        cy.get('#research-activities h2').text().should('not.be.empty');
+        cy.get('#research-activities .inner-panel h3').text().should('not.be.empty');
+        cy.get('#research-activities .inner-panel p').text().should('not.be.empty');
     })
 
     it('displays contact section', () => {
-        cy.expect('.contact h2').not.to.be.empty;
-        cy.expect('.contact p').not.to.be.empty;
-        cy.expect('.contact .feedback-content p').not.to.be.empty;
+        cy.get('#contacts h2').text().should('not.be.empty');
+        cy.get('#contacts .feedback-container p').text().should('not.be.empty');
     })
 
     it('displays footer', () => {
-        cy.expect('footer .footer-content li:first-child').not.to.be.empty;
+        cy.get('.footer-content li:first-child a').text().should('not.be.empty');
     })
-
```

Comment:
Can this be removed now?

Reply:
@robmint as Lukas suggested, just some commented out lines (38-41) that could be removed